plugins/modules/firewalld.py

```diff
@@ -84,6 +84,13 @@
     description:
       - Whether to run this module even when firewalld is offline.
     type: bool
+  target:
+    description:
+      - firewalld Zone target
+      - If state is set to C(absent), this will reset the target to default
+    choices: [ default, ACCEPT, DROP, REJECT ]
+    type: str
+    version_added: 0.2.0
```

Comment:
The next version would be 1.1.2 according to https://github.com/ansible-collections/ansible.posix/releases.
```suggestion
    version_added: 1.1.2
```